### DISCUSSION TITLE
Is AssumeRole API supported

### DISCUSSION THREAD

**Discussion by neepher** [minio/minio#16896] [Q&A] [answered]
After upgrade minio to RELEASE.2023-03-22T06-36-24Z with commit-id 035791669, the AssumeRole API call returns 200 OK but no STS credential returned.
Is this API no longer supported?

**Comment by vadmeste** [comment]
@neepher it should still be supported. What the error you are seeing ?

**Comment by neepher** [comment, marked as answer]
Oh, It's my fault. A wrong aws-cli profile is used to make the AssumeRole API call, the ak & sk is wrong.
However, a 401 response is expected if ak&sk is wrong, but 200 OK is responsed now, this is also confused.
